microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -378,6 +386,14 @@ Microsoft Edge doesn't store your geolocation coordinates. When making requests
 **InPrivate** browsing uses the precise location permission setting of the profile from which the **InPrivate** session was launched. **Guest** mode always asks you for permission before granting the site your precise location.
 
 
+<!-- ====================================================================== -->
+## History
+
+History now supports views to display images of your web navigations and enhance your history experience. All history data is stored locally on the device, and is organized per Microsoft Edge profile.  Images are saved for the duration the history exists (90 days).
+
+If you want Microsoft Edge to not capture images for your history, select the **Turn off images** button in the history hub.  Or, go to `edge://settings/privacy`, scroll down to the **Services** section, and then turn off the toggle for **Take Screenshots for History**.<!-- todo: in Dev 117 & Canary 117, there's instead a toggle labeled: **Save screenshots of site for History** -->  All previously saved images will be erased.
```

Comment:
In Dev 117 & Canary 117, in `edge://settings/privacy` > **Services** section, **Take Screenshots for History** is not found.  Instead there's a toggle labeled: **Save screenshots of site for History**.

Reply:
Style guide (I might tweak UI text, after confirming the UI):
* "go to": https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/n/navigate
* "toggle": https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/toggle

Reply:
replaced by fresh comment https://github.com/MicrosoftDocs/edge-developer/pull/2760/files#r1283903219